rpc/transport/transport.go

```diff
@@ -56,14 +56,36 @@ type Transport interface {
 }
 
 type OutgoingMessage struct {
-	Message rpccp.Message
-	Send    func() error
-	Release capnp.ReleaseFunc
+	Message  rpccp.Message
+	send     func() error
```

Comment:
How will Transport implementations defined outside of this package set send?

Reply:
D'oh!  Good catch.  Will fix this ASAP.